src/main/java/net/juniper/netconf/Main.java

```diff
@@ -0,0 +1,38 @@
+package net.juniper.netconf;
+
+import lombok.extern.slf4j.Slf4j;
+import org.xml.sax.SAXException;
+
+import java.io.IOException;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
+@Slf4j
+public class Main {
```

Comment:
This project is a library - it shouldn't have a Main class. Let alone one with hardcoded DNS names.